### PR TITLE
Update glossary.md

### PR DESCRIPTION
**Pull request by michaeldedrick**
## Motivation and Context

Fixes #
## Checklist:

- [ ] I have added a [Signed-off-by](https://github.com/hyperledger/fabric/blob/master/CONTRIBUTING.md#legal-stuff)
- [ ] Either no new documentation is required by this change, OR I added new documentation
- [ ] Either no new tests are required by this change, OR I added new tests
- [ ] I have run [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports), [go vet](https://golang.org/cmd/vet/), and [golint](https://github.com/golang/lint). I have cleaned up all valid errors and warnings in code I have added or modified. These tools may generate false positives. Don't be worried about ignoring some errors or warnings. The goal is clean, consistent, and readable code.

Signed-off-by:

### CODE REVIEW

**Comment by srderson** [comment]
Hi - realize this is a small change, but you still need to sign off on the PR for legal purposes.

**Comment by srderson** [comment]
Same fix as in PR #1569. Merged that one as it had a sign-off already.

File: docs/source/secured_private_asset_transfer_tutorial.rst [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+============================
+
+This tutorial will demonstrate how an asset can be represented and traded between organizations in a Hyperledger Fabric blockchain channel, while keeping details of the asset and transaction private.
+Each on-chain asset is a non-fungible token (NFT) that represents a specific asset having certain immutable metadata properties (such as size and color) with a unique owner. When the owner wants to sell the asset, both parties need to agree to the same price before the asset is transferred. The private asset transfer smart contract enforces that only the owner of the asset can transfer the asset. In the course of this tutorial, you will learn how Fabric features such as state based endorsement, private data, and access control come together to provide secured transactions that are both private and verifiable.
+
+This tutorial will deploy the [secured asset transfer sample](https://github.com/hyperledger/fabric-samples/tree/{BRANCH}/asset-transfer-secured-agreement/chaincode-go) to demonstrate how a transfer a private asset betwwen two organizations without publicly sharing data. You should have completed the task
```
Comment (denyeart) [inline docs/source/secured_private_asset_transfer_tutorial.rst]:
```suggestion
This tutorial will deploy the [secured asset transfer sample](https://github.com/hyperledger/fabric-samples/tree/{BRANCH}/asset-transfer-secured-agreement/chaincode-go) to demonstrate how to transfer a private asset between two organizations without publicly sharing data. You should have completed the task
```

File: docs/source/secured_private_asset_transfer_tutorial.rst [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+- A hash of each price agreement is stored on the ledger. The two hashes will match only if the two organizations have agreed to the same price. This allows the organizations to verify that they have come to agreement on the transfer details before the transfer takes place. A random trade id is added to the price agreement, which serves as a *salt* to ensure that other channel members can not use the hash on the ledger to guess the price.
+
+Transferring the asset
+---------------------------------------
+
+After the two organizations have agreed to the same price, the asset owner can use the transfer function to transfer the asset to the buyer:
+
+- Smart contract access control ensures that the transfer must be initiated by a member of the organization that owns the asset.
+- The transfer function verifies that the asset's private immutable properties passed to the transfer function matches the on chain hash if the asset data in private collection, to ensure that the asset owner is *selling* the same asset that they own.
```
Comment (denyeart) [inline docs/source/secured_private_asset_transfer_tutorial.rst]:
```suggestion
- The transfer function verifies that the asset's private immutable properties passed to the transfer function matches the on chain hash of the asset data in private collection, to ensure that the asset owner is *selling* the same asset that they own.
```

File: docs/source/secured_private_asset_transfer_tutorial.rst [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+We need to deploy an instance of the Fabric test network to run the smart contract. Open a command terminal and navigate to test network directory in your local clone of [fabric-samples](https://github.com/hyperledger/fabric-samples). We will operate from the `test-network` directory for the remainder of the tutorial.
+```
+cd fabric-samples/test-network
+```
+
+You can then deploy the network with the following command:
+```
+./network.sh up createChannel -c mychannel
+```
+The script will deploy the nodes of the network create a single channel named `mychannel` with Org1 and Org2 as channel members. We will use this channel to deploy the smart contract and trade our asset.
```
Comment (denyeart) [inline docs/source/secured_private_asset_transfer_tutorial.rst]:
```suggestion
The script will deploy the nodes of the network and create a single channel named `mychannel` with Org1 and Org2 as channel members. We will use this channel to deploy the smart contract and trade our asset.
```

File: docs/source/secured_private_asset_transfer_tutorial.rst [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+
+This tutorial will demonstrate how an asset can be represented and traded between organizations in a Hyperledger Fabric blockchain channel, while keeping details of the asset and transaction private.
+Each on-chain asset is a non-fungible token (NFT) that represents a specific asset having certain immutable metadata properties (such as size and color) with a unique owner. When the owner wants to sell the asset, both parties need to agree to the same price before the asset is transferred. The private asset transfer smart contract enforces that only the owner of the asset can transfer the asset. In the course of this tutorial, you will learn how Fabric features such as state based endorsement, private data, and access control come together to provide secured transactions that are both private and verifiable.
+
+This tutorial will deploy the [secured asset transfer sample](https://github.com/hyperledger/fabric-samples/tree/{BRANCH}/asset-transfer-secured-agreement/chaincode-go) to demonstrate how a transfer a private asset betwwen two organizations without publicly sharing data. You should have completed the task
+:doc:`install`.
+
+
+### Scenario requirements
```
Comment (denyeart) [inline docs/source/secured_private_asset_transfer_tutorial.rst]: This is markdown formatting.. and now that the tutorial is its own topic the section headers should all be reset to starting with single #.

**Pull request closed without merging**